WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -22,12 +22,12 @@ struct PointOfSaleDashboardView: View {
                         .ignoresSafeArea()
                 case .empty:
                     PointOfSaleItemListFullscreenView {
-                        PointOfSaleItemListEmptyView(base: .root)
+                        PointOfSaleItemListEmptyView(base: .root(.products))
```

Comment:
Yes, correct, for coupons, we'll need to have this logic update since we wouldn't want to show a full-screen error/empty view in full-screen.